src/main/java/org/dependencytrack/model/IntegrityAnalysisComponent.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Steve Springett. All Rights Reserved.
+ */
+package org.dependencytrack.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.IdGeneratorStrategy;
+import javax.jdo.annotations.PersistenceCapable;
+import javax.jdo.annotations.Persistent;
+import javax.jdo.annotations.PrimaryKey;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+/**
+ * Tracks integrity analysis results for components tracked by their uuid's.
+ *
+ * @author Meha Bhargava
+ */
+@PersistenceCapable(table = "INTEGRITY_ANALYSIS_COMPONENT")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class IntegrityAnalysisComponent {
+
+    @PrimaryKey
+    @Persistent(valueStrategy = IdGeneratorStrategy.NATIVE)
+    @JsonIgnore
+    private long id;
+    @Column(name = "COMPONENT_ID", allowsNull = "false")
+    @NotNull
+    private Component component;
+
+
+    public String getRepositoryIdentifier() {
+        return repositoryIdentifier;
+    }
+
+    public void setRepositoryIdentifier(String repositoryIdentifier) {
+        this.repositoryIdentifier = repositoryIdentifier;
+    }
+
+    /**
+     * This is a representation of the Package URL "name" field.
+     */
+    @Persistent
+    @Column(name = "REPOSITORY_IDENTIFIER", allowsNull = "false")
+    @NotNull
+    private String repositoryIdentifier;
+
+    /**
+     * The latest version of the component.
+     */
+    @Persistent
+    @Column(name = "MD5HASH_MATCHED", allowsNull = "false")
+    @NotNull
+    private String md5HashMatched;
```

Comment:
Should these be boolean fields ?
I would recommend moving fields to the top and getters setters below it to make it more readable.

isMd5HashMatched suggests that return type should be boolean